appholder/src/main/java/com/android/identity/wallet/util/NfcEngagementHandler.kt

```diff
@@ -103,24 +107,32 @@ class NfcEngagementHandler : HostApduService() {
 
         override fun onDeviceRequest(deviceRequestBytes: ByteArray) {
             log("Presentation Listener: OnDeviceRequest")
+            presentationLogStore.newLogEntryWithRequest(
```

Comment:
create a new log entry with specified request bytes, session transcript and NFC (static) engagement type -- this can be changed to `NFC_NEGOTIATED_HANDOVER` on future commits should the verifier request specific fields